babel.config.js

```diff
@@ -8,10 +8,10 @@ const pureDisplayNames = () => ({
       if (
         path.node.left.type === 'MemberExpression' &&
         path.node.left.property.name === 'displayName' &&
-        path.node.right.name
+        (path.node.right.name || path.node.right.extra?.raw)
       ) {
         const COMPONENT = path.node.left.object.name;
-        const DISPLAY_NAME = path.node.right.name;
+        const DISPLAY_NAME = path.node.right.name || path.node.right.extra?.raw;
```

Comment:
fixes this issue:
![image](https://github.com/radix-ui/primitives/assets/175330/5f82a14a-43eb-484a-806c-8dcb119861fe)